domains/Accounts/Controllers/AccountsLoginController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class AccountsLoginController extends Controller
+{
+
+    public function __invoke(Request $request): Response {
+
+        $this->validate($request, [
+            'email'    => ['required', 'string', 'email', 'max:255'],
+            'password' => ['required', 'string', 'min:6'],
+        ]);
+
+        $user = User::where('email', $request->email)->first();
+        if ($user) {
+            if (Hash::check($request->password, $user->password)) {
+                $token    = $user->createToken($user->email . '-'. Request()->ip())->accessToken;
+                $response = ['access_token' => $token];
+                return response($response, 200);
```

Comment:
To be aligned with the rest of the project, let's use `new Response(...)` instead of the `response()` helper.
Same for all other occurrences :)

Reply:
Thanks, changed!